server/server.js

```diff
@@ -0,0 +1,65 @@
+import express from "express";
+import cookieParser from "cookie-parser";
+import dotenv from "dotenv";
+import * as path from "path";
+import { isCorrectAnswer, Questions, randomQuestion } from "./questions.js";
+import bodyParser from "body-parser";
+
+dotenv.config();
+
+const app = express();
+app.use(bodyParser.json());
+app.use(cookieParser(process.env.COOKIE_SECRET));
+
+app.get("/quiz/score", (req, res) => {
+    const score = req.signedCookies.score
+        ? JSON.parse(req.signedCookies.score)
+        : {
+            answered: 0,
+            correct: 0,
+        };
+
+    res.json(score);
+});
+
+app.get("/quiz/random", (req, res) => {
+    const { id, question, answers } = randomQuestion();
+    res.json({ id, question, answers });
+});
+
+app.post("/quiz/answer", (req, res) => {
+    const { id, answer } = req.body;
+    const score = req.signedCookies.score
+        ? JSON.parse(req.signedCookies.score)
+        : {
+            answered: 0,
+            correct: 0,
+        };
+    const question = Questions.find((q) => q.id === id);
+    if (!question) {
+        return res.sendStatus(404);
+    }
+
+    score.answered += 1;
+    if (isCorrectAnswer(question, answer)) {
+        score.correct += 1;
+        res.cookie("score", JSON.stringify(score), { signed: true });
```

Comment:
Bra implementasjon av server-logikk, men dere burde brukt `express.Router` for å pakke det sammen